pkgs/servers/firebird/3.nix

```
@@ -0,0 +1,53 @@
+{ autoreconfHook
+, fetchFromGitHub
+, icu
+, lib
+, libedit
+, libtomcrypt
+, libtommath
+, stdenv
+, unzip
+, zlib
+, firebird
+
+, superServer ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "firebird";
+  version = "3.0.10";
+
+  src = fetchFromGitHub {
+    owner = "FirebirdSQL";
+    repo = "firebird";
+    rev = "v${version}";
+    sha256 = "sha256-PT2b3989n/7xLGNREWinEey9SGnAXShITdum+yiFlHY=";
+  };
+
+  buildInputs = [
+    icu
+    libedit
+    libtommath
+    zlib
+  ];
+
+  nativeBuildInputs = [ autoreconfHook ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook ];

  buildInputs = [
    icu
    libedit
    libtommath
    zlib
  ];
```
normally native is written first.

---

`nativeBuildInputs` is now first. Done.